### PR TITLE
feat(console): delete custom phrase

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

- delete custom phrase
- disallow to select selected language items

## Testing

delete-custom-phrases

### CODE REVIEW

File: packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.module.scss [hunk 1]
```diff
@@ -7,20 +7,24 @@
...
     color: var(--color-text);
+    display: flex;
+    justify-content: space-between;
 
-    > span {
-      margin-left: _.unit(2);
-      font: var(--font-body-medium);
-      color: var(--color-caption);
-    }
+    .languageInfo {
```
Comment (charIeszhao) [inline packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.module.scss:13]: Don't you need to set `.languageInfo` to flex and use `align-items: center` to center align the language name and built-in tag?

File: packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx [hunk 2]
```diff
@@ -21,8 +24,10 @@ type Props = {
 
 const LanguagesForm = ({ isManageLanguageVisible = false }: Props) => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
-  const { watch, control, register } = useFormContext<SignInExperienceForm>();
+  const { data: signInExperience } = useSWR<SignInExperience, RequestError>('/api/sign-in-exp');
+  const { watch, control, register, setValue } = useFormContext<SignInExperienceForm>();
   const isAutoDetect = watch('languageInfo.autoDetect');
+  const selectedDefaultLanguage = watch('languageInfo.fallbackLanguage');
```
Comment (charIeszhao) [inline packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx:30]: Do we have a task to rename `fallbackLanguage` to `defaultLanguage`?

File: packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.tsx [hunk 3]
```diff
@@ -24,10 +26,18 @@ const emptyUiTranslation = createEmptyUiTranslation();
...
+
+  const { languages, selectedLanguage, setIsDirty, setSelectedLanguage, stopAddingLanguage } =
+    useContext(LanguageEditorContext);
+
+  const [isDeletionAlertOpen, setIsDeletionAlertOpen] = useState(false);
 
   const isBuiltIn = isBuiltInLanguageTag(selectedLanguage);
 
+  const isDefaultLanguage =
+    signInExperience && signInExperience.languageInfo.fallbackLanguage === selectedLanguage;
```
Comment (charIeszhao) [inline packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.tsx:39]: nit: can be simplified
```suggestion
  const isDefaultLanguage = signInExperience?.languageInfo.fallbackLanguage === selectedLanguage;
```

File: packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx:30]: Cool, updated.

File: packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.module.scss (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.module.scss:13]: Cool, added.

File: packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx:30]: For compatibility reasons, we don't.

**Review by charIeszhao — APPROVED** [review]
LGTM

**Pull request merged**